the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
```

Comment:
I've been trying to remove generated headers from the app - can we remove these?

Reply:
Same for these